app/models/event.rb

```diff
@@ -0,0 +1,9 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: :User, foreign_key: :creator_id
```

Comment:
`foreign_key: :creator_id` not necessary here

Reply:
removed the foreign_key: :creator_id  as suggested